components/Layout.tsx

```diff
@@ -0,0 +1,15 @@
+import Header from './Header';
+import Footer from './Footer';
+import styles from '../styles/layout.module.css'
+
+const Layout = ({children}:{children: React.ReactNode}) => {
+    return (
+    <div className={styles.container}>
+      <Header/>
+      {children}
+      <Footer/>
+    </div>
+    );
+}
+
+export default Layout;
```

Comment:
```suggestion
const Layout = ({children}:{children: React.ReactNode}) => {
  return (
    <div className={styles.container}>
      <Header/>
      {children}
      <Footer/>
    </div>
  );
}

export default Layout;
```

Reply:
Could also be like this to shorten it up IIRC
```suggestion
const Layout = ({children}:{children: React.ReactNode}) => (
  <div className={styles.container}>
    <Header/>
    {children}
    <Footer/>
  </div>
);

export default Layout;
```